templates/search/locations-filter.php

```diff
@@ -0,0 +1,82 @@
+<div class="adwmpp-filter-wrapper adwmpp-locations-wrapper locations-wrapper adwmpp-dropdown dropdown">
+    <div class="adwmpp-input-wrapper">
+        <label><i class="adwmpp-locations-icon locations-icon"></i><?php _e('locations', ADMWPP_TEXT_DOMAIN); ?></label>
+        <span class="adwmpp-chosen-options chosen-options"></span>
+        <i class="adwmpp-toggle-arrow toggle-arrow"></i>
+    </div>
+    <div class="adwmpp-dropdown-wrapper dropdown-wrapper">
+        <div class="adwmpp-dropdown-body dropdown-body">
+            <?php if ($locations_filter_type == 'select') : ?>
+                <select class="admwpp-select admwpp-custom-select" name="location">
```

Comment:
Change the name to '**loc**' as using '**location**' will break the search page once you filter. _(did this happen on your end?)
Or maybe define it to the "queryVars" method and run some tests

Reply:
Done